packages/util-lang/lib/isInstanceInProtoChain.test.mjs

```diff
@@ -0,0 +1,8 @@
+import test from '@zorko-io/tool-test-harness'
+import {isInstanceInProtoChain} from './isInstanceInProtoChain'
+
+test('async - parses valid value', async (t) => {
+  // TODO: gh-82 fix existing test, provide few more use cases
```

Comment:
maybe change gh-82 because you closing this task by the PR?

Reply:
it's a bit different proposal, I want to not close the task after initial PR, so  @whiteadam2 can review it and continue that task. The main idea is to prep helper utility and gets unit-test practice